app/models/storage_location.rb

```diff
@@ -56,6 +56,10 @@ def item_total(item_id)
     inventory_items.select(:quantity).find_by(item_id: item_id).try(:quantity)
   end
 
+  def total_inventory_fair_market_value
+    items.sum(:value_in_cents)
```

Comment:
Can you use a safe-nav on the `value_in_cents` call? I think this will cause crashes if it's set to nil.
```suggestion
    items.sum(&:value_in_cents)
```

Reply:
Seconded. 

I think this is mergeable if you can get that in there.

Reply:
@mdworken I don't think that does safenav in this case -- when prefixing an argument in a method call, I believe it creates a proc.
ie.
```ruby
 items.sum(&:value_in_cents)
```
is equivalent to:
```ruby
 items.sum.map { |sum| sum.value_in_cents }
```